context.go

```diff
@@ -34,11 +34,12 @@ import (
 //	}
 //
 type Ctx struct {
-	WorkingDir string      // Where to execute.
-	GOPATH     string      // Selected Go path, containing WorkingDir.
-	GOPATHs    []string    // Other Go paths.
-	Out, Err   *log.Logger // Required loggers.
-	Verbose    bool        // Enables more verbose logging.
+	WorkingDir     string      // Where to execute.
+	GOPATH         string      // Selected Go path, containing WorkingDir.
+	GOPATHs        []string    // Other Go paths.
+	Out, Err       *log.Logger // Required loggers.
+	Verbose        bool        // Enables more verbose logging.
+	DisableLocking bool        // When set, disables locking.
```

Comment:
we need a little more on this comment here. it's in reference to a rather obscure feature that most people don't know about. with just this context, i would probably expect people to assume that it referred to, say, not writing out `Gopkg.lock` (which is not something we do at all). instead, maybe:

"When set, no lock file will be created to protect against multiple simultaneous dep processes."

Reply:
Absolutely.  Frankly I wrote something similar originally and changed my mind (something about that comment being much longer than the others -- silly I know).

I'll fix immediately.